compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -380,6 +380,9 @@ void addDoubleTilingPadExpertPassPipeline(OpPassManager &passManager) {
 
   {
     LinalgSingleTilingExpertPassOptions options;
+    // Explicitly set anchor op to matmul because only the op contains the
+    // reduction loops. Otherwise, it could try to tile tensor.pad ops.
+    options.anchorOpName = "linalg.matmul";
```

Comment:
This only works cause we only use this pipeline for `matmul` ops?

Reply:
Yes. This pipeline is only used by matmul ops.